raiden/tests/integration/network/proxies/test_payment_channel.py

```diff
@@ -101,28 +117,33 @@ def test_payment_channel_proxy_basics(
     assert channel_proxy_2.opened("latest") is True
 
     msg = "The channel was already closed, the second call must fail"
-    with pytest.raises(RaidenRecoverableError, message=msg):
+    with pytest.raises(RaidenRecoverableError):
         channel_proxy_1.close(
             nonce=0,
             balance_hash=EMPTY_HASH,
             additional_hash=EMPTY_HASH,
-            signature=EMPTY_SIGNATURE,
+            non_closing_signature=EMPTY_SIGNATURE,
+            closing_signature=LocalSigner(private_keys[1]).sign(data=closing_data),
             block_identifier=block_before_close,
         )
+        pytest.fail(msg)
 
     msg = "The channel is not open at latest, this must raise"
-    with pytest.raises(RaidenUnrecoverableError, message=msg):
+    with pytest.raises(RaidenUnrecoverableError):
         channel_proxy_1.close(
             nonce=0,
             balance_hash=EMPTY_HASH,
             additional_hash=EMPTY_HASH,
-            signature=EMPTY_SIGNATURE,
+            non_closing_signature=EMPTY_SIGNATURE,
+            closing_signature=LocalSigner(private_keys[1]).sign(data=closing_data),
             block_identifier="latest",
         )
+        pytest.fail(msg)
```

Comment:
why remove the msg from `raises()`? Due to the deprecation I guess: https://docs.pytest.org/en/latest/deprecations.html#raises-message-deprecated ?

Not a bad change but please keep changes to a bare minimum when making a PR. A PR should do one thing only and anything else is a distraction. Both for the author but also for the reviewer/s

Reply:
I didn't know `requirements/deps upgrade raiden-contracts` was possible.  Instead, I ran `requirements/deps upgrade`.  I'll fix this in this PR.